<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTrendbTable</class>
 <widget class="QWidget" name="QTrendbTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_in_out">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>491</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>显示设置</string>
   </property>
   <widget class="QComboBox" name="comboBox_pipeNum">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>23</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>26</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>当前组：</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>16</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>所有曲线属性相同</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_showAddr">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>112</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>控制地址：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ctlAddr">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../resourse.qrc">
      <normaloff>:/addressintput/images/address.bmp</normaloff>:/addressintput/images/address.bmp</iconset>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_showCondition">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>67</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>显示条件：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ctlAddr">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_lineWidth">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>80</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_showColor">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>110</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>显示颜色：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_lineWidth">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>80</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>线宽：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_lineStyle">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>52</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>线型：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_lineStyle">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
     <property name="icon">
      <iconset resource="../../resourse.qrc">
       <normaloff>:/lineType/images/LineType/L1.ico</normaloff>:/lineType/images/LineType/L1.ico</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
     <property name="icon">
      <iconset resource="../../resourse.qrc">
       <normaloff>:/lineType/images/LineType/L3.ico</normaloff>:/lineType/images/LineType/L3.ico</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
     <property name="icon">
      <iconset resource="../../resourse.qrc">
       <normaloff>:/lineType/images/LineType/L2.ico</normaloff>:/lineType/images/LineType/L2.ico</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
     <property name="icon">
      <iconset resource="../../resourse.qrc">
       <normaloff>:/lineType/images/LineType/L4.ico</normaloff>:/lineType/images/LineType/L4.ico</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
     <property name="icon">
      <iconset resource="../../resourse.qrc">
       <normaloff>:/lineType/images/LineType/L5.ico</normaloff>:/lineType/images/LineType/L5.ico</iconset>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_showColor">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>110</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>241</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>组设置</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>65</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>组数：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_groupSet">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>20</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>组设置</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_pipeCount">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>65</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_output">
   <property name="geometry">
    <rect>
     <x>269</x>
     <y>9</y>
     <width>231</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>显示范围</string>
   </property>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>31</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>最小：</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_outputMin">
    <property name="geometry">
     <rect>
      <x>84</x>
      <y>28</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-214748364.000000000000000</double>
    </property>
    <property name="maximum">
     <double>214748364.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>66</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>最大：</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_outputMax">
    <property name="geometry">
     <rect>
      <x>84</x>
      <y>63</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-214748364.000000000000000</double>
    </property>
    <property name="maximum">
     <double>214748364.000000000000000</double>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../resourse.qrc"/>
 </resources>
 <connections/>
</ui>
